lambda_function.py

```diff
@@ -0,0 +1,54 @@
+import json
+import requests
+
+def lambda_handler(event, context):
+
+    # Base URL of Gupshup's OTP API, including all constant params
+    baseURL = "https://enterprise.smsgupshup.com/GatewayAPI/rest?userid=2000173120&password=Avanti@12345&method=TWO_FACTOR_AUTH&v=1.1&msg=%25code%25%20is%20your%20OTP%20from%20Avanti%20Fellows&format=text&otpCodeLength=4&otpCodeType=NUMERIC";
+
+    response_headers={"Access-Control-Allow-Origin" : "*"}
+
+    # Check for query params, otherwise return 404 (Parameters not found)
+    if(event['queryStringParameters']):
```

Comment:
rather than doing this:
```python
if X:
     if Y:
         return A
    return B
return C
```

do this:

```python

if not X:
   return C

if not Y:
    return B

return A
```

this avoids making the code branched - as in, prevents the width of the code from increasing.